.github/workflows/integration.yml

```diff
@@ -68,7 +94,7 @@ jobs:
         run: |
           # https://minikube.sigs.k8s.io/docs/handbook/pushing/#7-loading-directly-to-in-cluster-container-runtime
           minikube image load apache/celeborn:latest
-          helm install celeborn charts/celeborn -f ./tests/kubernetes-it/docker/helm/values.yaml
+          helm install celeborn charts/celeborn -f charts/celeborn/ci/values.yaml
```

Comment:
if we publish the chart to habor, will the values.yaml under ci be published too? does it have side-effects?

Reply:
> if we publish the chart to habor, will the values.yaml under ci be published too? does it have side-effects?

@pan3793 I will add a `.helmignore` file so that we can specify patterns for files and directories that should be ignored when packaging and publishing the chart. And it does not have any side-effects.